v2/googlecloud-to-elasticsearch/src/main/java/com/google/cloud/teleport/v2/elasticsearch/transforms/EventMetadataBuilder.java

```diff
@@ -22,6 +22,8 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.node.ObjectNode;
 import com.google.cloud.teleport.v2.elasticsearch.options.PubSubToElasticsearchOptions;
+import com.google.cloud.teleport.v2.elasticsearch.utils.ElasticsearchUtils;
+
```

Comment:
@Cherepushko -  Can you remove extra space between imports since this is causing stylecheck violation
[ERROR] src/main/java/com/google/cloud/teleport/v2/elasticsearch/transforms/EventMetadataBuilder.java:[27] (imports) ImportOrder: Extra separation in import group before 'java.io.Serializable'

Reply:
Done